test/test_opportunity/test_opportunity.cpp

```diff
@@ -1,18 +1,24 @@
 #include "../test_util.hpp"
 #include <unity.h>
+#include <time.h>
+#include <stdio.h>
 
 extern "C"
 {
 #include <opportunity.h>
 }
 
+#define RANDOM_SEED time(0)
```

Comment:
Since this macro is calling a function, it would be better if it were called GET_RANDOM_SEED or something. Otherwise it's a bit unclear—it looks like a constant but it's actually different every time you call it.

Reply:
It's different every time you call it, but it's constant throughout the tests, so wouldn't `RANDOM_SEED` be clearer since  it's the same random seed throughout the tests?